src/main/java/core/basesyntax/AddFruit.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.Operation;
+import java.util.List;
+
+public class AddFruit implements Operation {
+
+    public List<Fruit> operation(List<Fruit> fruitsAvailable, Transaction fruitsFromFile) {
```

Comment:
please check all your classes - while class names should be nouns, method names should be **verbs**

Reply:
done